### ISSUE TITLE
New Maintainer?

### ISSUE THREAD

**Issue by pattyland** [k0rventen/avea#1]
Hi,

since a few weeks I got emails from users of a project from me, that is using the avea library on PyPI, that version [1.0.4](https://pypi.org/project/avea/1.0.4/) is missing.

Now it seems like it got deleted? Even the [author on GitHub](https://github.com/Hereath/) and all his projects got deleted? Did he assign the package on PyPI to you? I'm a bit confused

Greets, Sören

**Comment by k0rventen** [comment]
Hi pattyland,

I'm the original author of avea.
For security reasons I deleted my old GitHub account a few weeks back, and had to reopen a new one (with a new nickname). I'm still in the process of reuploading all of my repos, including avea. I've also updated the pip package, but as it was not linked to any username, it's still [available here](https://pypi.org/project/avea/), v1.2.6 as of today.

To resolve you email issues I guess you would have to delete your avea package and refork mine, so future issues won't be assigned to you.

Sry for the confusion,
Kr, Corentin
